interceptors/Mementifier.cfc

```diff
@@ -132,6 +132,8 @@ component {
 		if ( isSimpleValue( arguments.excludes ) ) {
 			arguments.excludes = listToArray( arguments.excludes );
 		}
+		arguments.includes = duplicate( arguments.includes );
```

Comment:
I'm going to defer to @lmajano 's thoughts on this, but since we are mutating these two args within the array and the remaining processing is all encapsulated in to a single function, it feels like re-scoping this duplicate to local/var and then using that reference, through the remaining routines would serve two purposes:

1.  It would encapsulate all subsequent mutations within the scope of the function
2. The original includes argument would be preserved - which would allow us to reference it, if it ever became necessary down the road.

The only thing that would be necessary for this would be to change to a `var` scope for the two duplications, and then change all references from `arguments` to `local` in the function body.